README.md

```diff
@@ -9,27 +15,31 @@ As EPIC Andelans who work towards advancing human potential and giving back to t
 we wish to empower others by building a web app that helps people exchange
 messages/information over the internet.
 
-## Features
+## UI Features
 
-* As a User:
-    1. I should be able to signup for a new EPIC Mail account.
-    2. I should should be able to sign into my EPIC Mail account.
-    3. I should be able to reset my password
-    4. I should be able to send email to individuals or groups.
-    5. I should be able to view my inbox.
-    6. I should be able to read an email message and any resulting thread.
-    7. I should be able to view sent messages and retract a sent message.
+ |`/user`| `/admin`|
+ |---|---|
+ |- signup for a new EPIC Mail account| - create groups|
+ |- sign into my EPIC Mail account| - add individuals to groups|
+ |- reset password||
+ |- view inbox||
+ |- reade email message||
+ |- view sent messages and retract a sent message||
+ |- send email to individuals or groups||
 
-* As an Admin:
-    1. I should be able to create a group and add different users.
+ ## login details
+
+|user account|Admin account|
+|:---:|:---:|
+|email `ivan@gmail.com`||
+|password `brenda`||
```

Comment:
The `Installation` instructions can be better. 
Reach out to Jenny or Ahmed

Reply:
I will have the changes in the next push to this PR to update the requested changes and Readme.